resources/views/layouts/app.blade.php

```diff
@@ -9,12 +9,12 @@
     @yield ('meta')
 
     <link href="/css/app.css" rel="stylesheet">
-    @if (isset($gistlog) && $gistlog->isSecret())
+    @if (isset($gistlog) && $gistlog->isSecret() || !\App::environment('production'))
     <meta name="robots" content="noindex, nofollow">
     @endif
 
     <!-- Fonts -->
-    <link rel="stylesheet" href="//brick.a.ssl.fastly.net/Roboto:300,400,700/Linux+Libertine:400,400i,700,700i">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i" rel="stylesheet">
```

Comment:
Why moving to the google fonts URL? You can see we have roboto here on the landing page:

         <link rel="stylesheet" href="//brick.a.ssl.fastly.net/Roboto:300,400,700">

Also: Do we really need all those styles? If so, that's cool. Just trying to keep the weight down.